tasks/managed/collect-atlas-params/README.md

```diff
@@ -8,6 +8,16 @@ skipped.
 
 ## Parameters
 
-| Name     | Description                                                                                                    | Optional | Default value |
-|----------|----------------------------------------------------------------------------------------------------------------|----------|---------------|
-| dataPath | Path to the merged data JSON file generated by collect-data task and containing the Atlas configuration option | No       | -             |
+| Name                    | Description                                                                                                                | Optional | Default value                                             |
+|-------------------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------------------------------------------------|
```

Comment:
missing orasOptions and trustedArtifactsDebug

Reply:
fixed